edi_oca/models/edi_exchange_consumer_mixin.py

```diff
@@ -232,3 +238,10 @@ def get_edi_access(self, doc_ids, operation, model_name=False):
         else:
             check_operation = operation
         return check_operation
+
+    def _edi_set_origin(self, exc_record):
+        self.sudo().update({"origin_exchange_record_id": exc_record.id})
+
+    def _edi_get_origin(self):
+        self.ensure_one()
+        return self.origin_exchange_record_id
```

Comment:
Any specific reason for having setter/getter methods for this field?

Reply:
they are not setter/getter for a field, they are api methods. And no that you let me think about them, they should not be private.